kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -317,6 +316,7 @@ impl<'tcx> GotocCtx<'tcx> {
 
     fn codegen_direct_literal(&mut self, ty: Ty<'tcx>, init: Expr) -> Expr {
         let func_name = format!("gen-{}:direct", self.ty_mangled_name(ty));
+        let pretty_name = format!("init_direct::<{}>", self.ty_pretty_name(ty));
```

Comment:
How did you come up with this name?

Reply:
I guessed that this was an intiialization function.